test/test_um2netcdf.py

```diff
@@ -674,3 +675,46 @@ def test_fix_cell_methods_keep_weeks():
     assert mod.method == cm.method
     assert mod.coord_names == cm.coord_names
     assert mod.intervals[0] == "week"
+
+
+LEVEL_HEIGHTS = [20.0003377]  # NB: technically only need [0]
```

Comment:
I'm wondering whether it's worth noting that this value comes from the `zsea_at_theta` data, so that it's clearer in `test_fix_level_coord_theta` why we are expecting the names to have the `theta` added. This might get a bit confusing though as the same data is used in the `test_fix_level_coord_rho` test, so am ok with either way here.

Reply:
Yes, the provenance should be mentioned. Also, the `z_theta` fixture has `data = np.array([0.0, 20.000337706971997, 80.0...` where the `20.0...` is `data[1]` & not `data[0]` as per the comment. It's small, but indicates forgetting a detail within a few days.

Reply:
I've made the `[0]` comment clearer. The test funcs need `points[0]`, so the incoming data only needs one point.